app/Http/Controllers/ReviewController.php

```diff
@@ -152,4 +152,11 @@ public function destroy($id)
 
         return redirect()->back()->with('status', trans('msg.delete_successful'));
     }
+
+    public function history()
+    {
+        $reviews = Review::where('user_id', Auth::user()->id)->paginate(config('default.pagination'));
```

Comment:
```suggestion
        $reviews = Auth::user()->reviews()->paginate(config('default.pagination'));
```

Reply:
![image](https://user-images.githubusercontent.com/56680337/95964820-6ecd4780-0e33-11eb-95a9-6a49cd26556a.png)
Trước e định làm thế mà nó toàn hiện lỗi ntn ý chị, code vẫn đúng cơ mà vsc nó cứ báo thế